bears/general/InvalidLinkBear.py

```diff
@@ -80,19 +81,25 @@ def find_links_in_file(file, network_timeout, link_ignore_regex,
                                         # Unbalanced parenthesis
             (?<!\.)(?<!,)               # Exclude trailing `.` or `,` from URL
             """, re.VERBOSE)
+        xml_namespace_links = set()
```

Comment:
I'm sure there will be other cases like this. Create an enum for 'first context' of each url, and start it with only 'xml namespace'.